deploy/examples/example-custom-build-images.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoApp
+metadata:
+  name: example-quarkus
+spec:
+  build:
+    #env:
+    # this value is just an example, set yours (optional)
+    #- name: MAVEN_MIRROR_URL
+    #  value: "http://nexus3.kogito-tools.svc.cluster.local:8081/repository/maven-public/"
+    gitSource:
+      contextDir: jbpm-quarkus-example
+      uri: 'https://github.com/kiegroup/kogito-examples'
+      reference: 0.7.0-rc4
+    # images that don't have a custom tag, will use this version
+    imageVersion: 0.7.0-rc4
+    # the image runtime build will use this image
+    imageRuntimeTag:  quay.io/ricardozanini/kogito-quarkus-jvm-ubi8:0.7.0-rc1
```

Comment:
maybe use kiegroup and tag 0.7.0.rc4?

Reply:
The tag is already used by the `version`. The point here is to have a different namespace, so others can understand that they might have their own image in their own namespace.